keyboards/dtd/ldmlKeyboard3.dtd

```diff
@@ -17,6 +17,8 @@ Please view the subcommittee page for the most recent information.
 <!ATTLIST keyboard3 conformsTo (techpreview) #REQUIRED >
     <!--@MATCH:any-->
     <!--@METADATA-->
+<!ATTLIST keyboard3 xmlns CDATA #IMPLIED >
+    <!--@METADATA-->
```

Comment:
Should add appropriate MATCH annotations. Should be separate ticket, though.

Reply:
maybe `<!--@MATCH:any-->`, but we shouldn't specify it further.  The default namespace is specified in https://www.w3.org/TR/xml-names/#defaulting and should not need to be specified here.

Reply:
done